tests/Tests/100__deploy/100__installation/102__post_install.robot

```diff
@@ -271,7 +271,7 @@ Test Setup For Rhods Dashboard
 
 Test Teardown For Configmap Changed On RHODS Dashboard
     [Documentation]    Test Teardown for Configmap changes on Rhods Dashboard
-    Oc Patch    kind=ConfigMap      namespace=redhat-ods-applications    name=rhods-enabled-applications-config    src={"data": {"jupyterhub": "true"}}   #robocop: disable
+    Oc Patch    kind=ConfigMap      namespace=redhat-ods-applications    name=odh-enabled-applications-config    src={"data": {"jupyterhub": "true"}}   #robocop: disable
```

Comment:
shouldn't you change  from  src={"data": {"jupyterhub": "true"}} to src={"data": {"jupyterhub": "true", "jupyter": "true"}} for KFNBC

Reply:
I think it should contain only  "jupyter": "true" now that we are going to merge kfnbc branch

Thanks for catching it

Reply:
What i have observed during migration is that it contains both 'jupyterhub` and `jupyter` in the configmap. It shouldn't contain both it should only contain jupyter

Reply:
it's a bug/question to raise to developers about migration script

Reply:
even in fresh cluster you can observe this, I think we can open a bug in downstream. Regarding migration i have asked developer in git

Reply:
I will open the bug

Reply:
nope